File: app/src/main/java/br/com/digitalhouse/projetomarvel/api/Item.java

package br.com.digitalhouse.projetomarvel.api;

import com.google.gson.annotations.Expose;

@SuppressWarnings("unused")
public class Item {

    @Expose
    private String name;
    @Expose
    private String resourceURI;
    @Expose
    private String role;
    @Expose
    private String type;

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getResourceURI() {
        return resourceURI;
    }

    public void setResourceURI(String resourceURI) {
        this.resourceURI = resourceURI;
    }

    public String getRole() {
        return role;
    }

    public void setRole(String role) {
        this.role = role;
    }

    public String getType() {
        return type;
    }

    public void setType(String type) {
        this.type = type;
    }

}

File: app/src/main/java/br/com/digitalhouse/projetomarvel/api/Item.kt
package br.com.digitalhouse.projetomarvel.api

import com.google.gson.annotations.Expose

class Item {
    @Expose
    var name: String? = null

    @Expose
    var resourceURI: String? = null

    @Expose
    var role: String? = null

    @Expose
    var type: String? = null

}